pkgs/by-name/av/av1an/package.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, symlinkJoin, makeBinaryWrapper, testers, nix-update-script
+, av1an-unwrapped, av1an, ffmpeg, python3Packages, libaom, svt-av1, rav1e
+, libvpx, x264, x265, libvmaf, withAom ? true, # AV1 reference encoder
+withSvtav1 ? false, # AV1 encoder/decoder (focused on speed and correctness)
+withRav1e ? false, # AV1 encoder (focused on speed and safety)
+withVpx ? true, # VP8 & VP9 de/encoding
+withX264 ? true, # H.264/AVC encoder
+withX265 ? true, # H.265/HEVC encoder
+withVmaf ? false, # Perceptual video quality assessment algorithm
+}:
```

Comment:
same as above, one argument per line.  additionally, try not to mix the comma-first and comma-last styles